test/e2e/networking.go

```diff
@@ -299,7 +299,7 @@ func LaunchNetTestPodPerNode(f *Framework, nodes *api.NodeList, name, version st
 	return podNames
 }
 
-func getReadyNodes(f *Framework) (nodes *api.NodeList, err error) {
+func GetReadyNodes(f *Framework) (nodes *api.NodeList, err error) {
```

Comment:
maybe move these to util.go, since that's where we put most of the "generally useful" functions?

also, these should get docstrings now that they're exported.